be1-go/cli/mod.go

```diff
@@ -131,14 +110,14 @@ func Serve(cliCtx *cli.Context, user string) error {
 	return nil
 }
 
-// connectToSocket establishes a connection to another server's witness
+// connectToSocket establishes a connection to another server's server
 // endpoint.
-func connectToSocket(otherHubType hub.HubType, address string, h hub.Hub,
+func connectToSocket(address string, h hub.Hub,
 	wg *sync.WaitGroup, done chan struct{}) error {
 
 	log := popstellar.Logger
 
-	urlString := fmt.Sprintf("ws://%s/%s/witness", address, otherHubType)
+	urlString := fmt.Sprintf("ws://%s/sockets/server", address)
```

Comment:
why not just the following? 😄 
```suggestion
	urlString := fmt.Sprintf("ws://%s/server", address)
```